_i18n/en/develop.md

```diff
@@ -0,0 +1,34 @@
+
+## Where we need help
+There's a few areas where we could well use your support. Because they're important, often-requested, or both. Get yourself into any of the following issues and we'll add you to our (mental) list for eternal gratitude:
+* [Re-implementation of Chromecast support](https://forum.antennapod.org/t/developers-wanted-google-chromecast-support/657) (otherwise in May 2022 we'll unfortunately have to depricate it)
+* [Improvements to and new features for tagging podcasts](https://github.com/AntennaPod/AntennaPod/issues/1711) (issue closed, but many valuable requests & ideas can be found there still), including [user-definable queues](https://github.com/AntennaPod/AntennaPod/issues/2648)
+* [Option to download oldest episodes first](https://github.com/AntennaPod/AntennaPod/issues/1077) and other issues around [automatic deletion](https://github.com/AntennaPod/AntennaPod/milestone/47)
+* [Podcasting 2.0 and Podcast Index integration](https://github.com/AntennaPod/AntennaPod/labels/Podcast%20Index%20%2F%20Podcasting%202.0)
+
+Please don't just submit a PR, but always announce your interest first. Sometimes we didn't yet reach an agreement on a user experience and interface. It can happen that someone agreed to start working on something, but we forgot to 'assign' the GitHub issue. Chipping in on our forum is the best way to avoid lost hours on code that won't be accepted.
+
+Apart from these major projects, you can always check out the issues with the ['help wanted' label](https://github.com/AntennaPod/AntennaPod/labels/help%20wanted) on GitHub.
+
+## Bug fixing
+Users often chip in on GitHub and our forum with possible bugs. Your go-to-list for a weekend bug-hunt is the ['confirmed bugs' issue label](https://github.com/AntennaPod/AntennaPod/labels/confirmed%20bug) on GitHub.
+
+## Getting started
+To read up on how we develop the app, please have a look at the [GitHub wiki](https://github.com/AntennaPod/AntennaPod/wiki). There you'll find information about code style, build instructions and debug logs.
+
+# Website
+The base of our new website was kindly built by [Anxhelo](https://lushka.al). Now we're ready to expand! Are you familiar with [Jakyll](https://jekyllrb.com/)? Then help us implement (parts of) [our website wishlist](https://forum.antennapod.org/t/sitemap-for-the-new-website/240) and check out our [website repository](https://github.com/AntennaPod/antennapod.github.io) now.
+
+# Get in touch
+Vaguely interested but not sure yet? Ready to get started? Get in touch with @Keunes or @ByteHamster on [our forum](https://forum.antennapod.org)!
+
+# Upstream & other projects
+AntennaPod doesn't live in isolation. The project relies on many others - contributing to those means helping AntennaPod thrive:
+* **Nextcloud Podcast app** (Vue, PHP, Javascript)
```

Comment:
As the app does not support AntennaPod yet, I would not list it here yet.

Reply:
The point here is exactly to get the app to support AntennaPod. Given the issues with gpodder.net synchronisation, I think it's great if we can focus attention to (& get implementation of) a viable alternative. Better for us, our users, and gpodder (less pressure on their servers). The NC podcast app initiator clearly has interest, as they had added it previously to the first next milestone, and then set it to 'future' milestone.